src/components/List/src/listitem.module.css

```diff
@@ -0,0 +1,82 @@
+.denhaag-list__item {
+  background-color: white;
+  padding-block: var(--denhaag-listitem-padding-block);
+  padding-inline: var(--denhaag-listitem-padding-inline);
+}
+
+.denhaag-list__item:focus {
+  border-color: var(--denhaag-listitem-focus-border-color);
+  border-style: var(--denhaag-listitem-focus-border-style);
+  border-width: var(--denhaag-listitem-border-size);
+  padding-block: calc(var(--denhaag-listitem-padding-block) - var(--denhaag-listitem-border-size));
+  padding-inline: calc(var(--denhaag-listitem-padding-inline) - var(--denhaag-listitem-border-size));
+}
+
+.denhaag-list__item:active {
+  border: var(--denhaag-listitem-border-size) solid var(--denhaag-blue-3);
+  padding-block: calc(var(--denhaag-listitem-padding-block) - var(--denhaag-listitem-border-size));
+  padding-inline: calc(var(--denhaag-listitem-padding-inline) - var(--denhaag-listitem-border-size));
+}
+
+.denhaag-list__item:active > .denhaag-list__secondaryaction {
+  right: calc(var(--denhaag-listitem-secondaryaction-right) - var(--denhaag-listitem-border-size));
+}
+
+.denhaag-list__item:focus > .denhaag-list__secondaryaction {
+  right: calc(var(--denhaag-listitem-secondaryaction-right) - var(--denhaag-listitem-border-size));
+  color: var(--denhaag-blue-3);
```

Comment:
Don't refer to proprietary styles in the component css. It's probably a good idea to create a component design token for the font color, and giving it a default value of --denhaag-blue-3.